api/proto/teleport/integration/v1/awsoidc_service.proto

```diff
@@ -22,6 +22,10 @@ option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport
 
 // AWSOIDCService provides access to AWS APIs using the AWS OIDC Integration.
 service AWSOIDCService {
+  // ListEICE returns a list of EC2 Instance Connect Endpoints.
+  // An optional NextToken that can be used to fetch the next page.
+  rpc ListEICE(ListEICERequest) returns (ListEICEResponse);
```

Comment:
nit: Might be worth adding what is the source of this, much like the `ListDatabases` call below lists some APIs.